### PR TITLE
Custom form modals

### PR DESCRIPTION
**Pull request by CallToPower**

### CODE REVIEW

File: source/components/ui/msm-modal/msm-modal-form.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="modal-header">
+  <h3 class="modal-title">{{ vm.title | translate }}</h3>
+  <span class="modal-close" ng-click="vm.onDismiss ? vm.onDismiss() : $dismiss('cancel')"><i class="zmdi zmdi-close img-close"></i></span>
+</div>
+<div class="modal-body modal-mobile-show">
```
Comment (fynnfeldpausch) [inline source/components/ui/msm-modal/msm-modal-form.html]: Missing... Should this be merged to `<div class="modal-body"></div>`?

Comment (CallToPower) [inline source/components/ui/msm-modal/msm-modal-form.html]: Forgot about mobile... one second.

File: source/components/ui/msm-modal/msm-modal-form.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <div>
+    <form name="form" id="changePassword-form" role="form">
+      <formly-form model="vm.model" fields="vm.fields" options="vm.options" form="vm.form">
+      </formly-form>
+    </form>
+  </div>
+</div>
+<div class="modal-footer">
+  <button ng-repeat="button in vm.buttons" class="btn btn-{{ button.context }}" data-ng-disabled="{{button.constraint}}"
+          ng-class="{ 'modal-mobile-hide': button.hideMobile }" ng-click="button.onClick()">
```
Comment (fynnfeldpausch) [inline source/components/ui/msm-modal/msm-modal-form.html]: missing `'btn-zmdi'` option here... see default template (not select template)

File: source/components/ui/msm-modal/msm-modal-form.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <i ng-if="button.icon" class="zmdi zmdi-{{ button.icon }}"></i>
```
Comment (fynnfeldpausch) [inline source/components/ui/msm-modal/msm-modal-form.html]: missing `'zmdi-hc-fw'` class here... see default template (not select template)

**Pull request merged**